pointgrey_camera_driver/src/PointGreyCamera.cpp

```diff
@@ -988,7 +1004,7 @@ void PointGreyCamera::grabImage(sensor_msgs::Image &image, const std::string &fr
     uint8_t bitsPerPixel = rawImage.GetBitsPerPixel();
 
     // Set the image encoding
-    std::string imageEncoding = sensor_msgs::image_encodings::MONO8;
+    std::string imageEncoding = sensor_msgs::image_encodings::RGB8;
```

Comment:
This is not a breaking change, it is just the first initialization, I reverted it.